connectors/sources/jira.py

```diff
@@ -216,65 +220,92 @@ def get_default_configuration(cls):
                 "label": "Jira Server password",
+                "sensitive": True,
+                "order": 3,
                 "type": "str",
+                "value": "changeme",
             },
             "service_account_id": {
-                "value": "me@example.com",
+                "depends_on": [{"field": "data_source", "value": JIRA_ONLINE}],
                 "label": "Jira Cloud service account id",
+                "order": 4,
                 "type": "str",
+                "value": "me@example.com",
             },
             "api_token": {
-                "value": "abc#123",
+                "depends_on": [{"field": "data_source", "value": JIRA_ONLINE}],
                 "label": "Jira Cloud API token",
+                "order": 5,
+                "sensitive": True,
                 "type": "str",
+                "value": "abc#123",
             },
             "host_url": {
-                "value": "http://127.0.0.1:8080",
                 "label": "Jira host url",
+                "order": 6,
                 "type": "str",
+                "value": "http://127.0.0.1:8080",
             },
             "projects": {
-                "value": "*",
+                "display": "textarea",
                 "label": "Jira Project Keys",
                 "type": "list",
+                "value": "*",
             },
             "ssl_enabled": {
-                "value": False,
-                "label": "Enable SSL verification (true/false)",
+                "display": "toggle",
+                "label": "Enable SSL verification",
+                "order": 7,
                 "type": "bool",
+                "value": False,
             },
             "ssl_ca": {
-                "value": "",
+                "depends_on": [{"field": "ssl_enabled", "value": True}],
                 "label": "SSL certificate",
+                "order": 8,
                 "type": "str",
-            },
-            "enable_content_extraction": {
-                "value": True,
-                "label": "Enable content extraction (true/false)",
-                "type": "bool",
+                "value": "",
             },
             "retry_count": {
-                "value": 3,
+                "default_value": 3,
+                "display": "numeric",
                 "label": "Maximum retries for failed requests",
+                "order": 9,
+                "required": False,
                 "type": "int",
+                "ui_restrictions": ["advanced"],
+                "value": 3,
             },
             "concurrent_downloads": {
-                "value": 50,
+                "default_value": 50,
+                "display": "numeric",
                 "label": "Number of concurrent downloads for fetching attachment content",
+                "order": 10,
+                "required": False,
                 "type": "int",
+                "ui_restrictions": ["advanced"],
```

Comment:
Just curious, what does `ui_restrictions` control and what other options are there?

Reply:
Oh Gotcha, Thanks!